website/docs/r/ons_instance.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_ons_instance"
+sidebar_current: "docs-alicloud-resource-ons-instance"
+description: |-
+  Provides a Alicloud ONS Instance resource.
+---
+
+# alicloud\_ons\_instance
+
+Provides a ONS instance resource.
+
+-> **NOTE:**  The number of instances in the same region cannot exceed 8. Does not support region "mq-internet-access" and "ap-southeast-5". 
```

Comment:
The number of instances in the same region cannot exceed 8. At present, the resource does not support region "mq-internet-access" and "ap-southeast-5".